tsl/test/sql/include/cagg_on_cagg_common.sql

```diff
@@ -6,31 +6,74 @@
 CREATE MATERIALIZED VIEW :CAGG_NAME_1ST_LEVEL
 WITH (timescaledb.continuous, timescaledb.materialized_only=true) AS
 SELECT
+\if :IS_JOIN
+  time_bucket(:BUCKET_WIDTH_1ST, "time") AS bucket,
+  SUM(temperature) AS temperature,
+  device_id
+\else
```

Comment:
Insted of all this conditionals what about something like this in the tests that include `cagg_on_cagg_common.sql`
```sql
-- In the cagg_on_cagg.sql
\set EXTRA_GROUPBY_COLUMN
-- In the cagg_on_cagg_joins.sql
\set EXTRA_GROUPBY_COLUMN ,device_id
```

And then use it in the query in `cagg_on_cagg_common.sql`:
```suggestion
  time_bucket(:BUCKET_WIDTH_1ST, "time") AS bucket,
  SUM(temperature) AS temperature
  :EXTRA_GROUPBY_COLUMN
```
Doing that you'll reduce the log output.